backend/app/core/actions/microsoft_teams_action.py

```diff
@@ -0,0 +1,59 @@
+from app.models.destinations.destination import MicrosoftTeams
+from app.settings import settings
+import apprise
+from apprise import Apprise
+import pathlib
+
+
+class MicrosoftTeamsAction:
+    def notify(self, action: MicrosoftTeams, action_type: str, **kwargs: dict) -> tuple[Apprise, str, str]:
+        """
+        Send a notification to Email
+        :param action: Email action
+        :param action_type: Action type e.g. validation
+        :param kwargs: Such as, GE Validation
+        :return: True if the notification was sent successfully
+        """
+        if action_type == "validation":
+            title, body = self.get_validation(kwargs["validation"])
+        else:
+            raise NotImplementedError(f"action_type '{action_type}' is not implemented.")
+
+        teams_template_dir = str(pathlib.Path(__file__).parent.resolve())
+        template_path = f"{teams_template_dir}/microsoft_teams_template.json"
+
+        ar = apprise.Apprise()
+        ar.add(f"{action.webhook.get_decrypted_value()}?template={template_path}")
```

Comment:
I don't really know how Apprise works but, from what I understand here, we are passing to the webhook a local file path to the template. Will it really work in production? A remote server won't be able to read this file on the local filesystem unless properly served by an HTTP server 🤔

Reply:
I would expect it to fail in the Docker container then, wouldn't you?

Reply:
Sorry, I misread your message. Apprise uses the local directory to receive the file template not the remote server. `microsoft_teams_template.json` will be in the Docker container when deployed and Apprise will access it from the Docker container.

Reply:
Ok, that answers my question 👍